server/publications/users.js

```diff
@@ -9,3 +8,11 @@ Meteor.publish('user-miniprofile', function(userId) {
     },
   });
 });
+
+Meteor.publish('user-admin', function() {
```

Comment:
No, this is not that kind of publication, it's more like user-is-admin, but definitely not admin-users,